fluent-syntax/test/fixtures_behavior/variant_lists.ftl

```diff
@@ -0,0 +1,41 @@
+#~ ERROR E0014, pos 16
+message1 =
+    {
+        *[one] One
+    }
+
+#~ ERROR E0014, pos 59
+message2 =
+    {
+        *[one] {
+            *[two] Two
+         }
+    }
```

Comment:
This test doesn't seem to do anything beyond message1, AFAICT. At least, it dies with the same error message on the same code point.

Reply:
Good point, I'll remove it.